docs/advisors/checks/mongodb-swap-allocation.md

```diff
@@ -0,0 +1,39 @@
+# MongoDB Swap Allocation
+
+## Description
+This check warns if there is no swap memory allocated to your instance.
+
+## Resolution
+
+MongoDB performs best where swapping can be avoided or kept to a minimum since retrieving data from swap on disk will always be slower than accessing data in RAM. However, if the system hosting MongoDB runs out of RAM, swapping can prevent the Linux OOM Killer from terminating the mongod process.
```

Comment:
Should we move context information to the Description section? Resolution section should only contain info on how to fix the issue reported by the check.